tests/test_toa.py

```diff
@@ -87,19 +87,17 @@ def test_mulit_obs(self):
 
         toas = TOAs(toalist=[t1, t2, t3])
 
-        # Table will be grouped by observatories, and will be sorted by the
-        # observatory so the TOA order will be different
-        assert toas.table["obs"][0] == site2.name
-        assert toas.table["mjd"][0] == t2.mjd
-        assert toas.table["obs"][1] == site3.name
-        assert toas.table["mjd"][1] == t3.mjd
-        assert toas.table["obs"][2] == site1.name
-        assert toas.table["mjd"][2] == t1.mjd
+        assert toas.table["obs"][0] == site1.name
```

Comment:
What property is this test verifying? That the order of the result is just the concatenation of the originals? I don't understand what makes this the right behaviour.